CHANGELOG.adoc

```diff
@@ -135,6 +135,7 @@ The importer can now import SysML files that contain `FeatureChainingExpression`
 - https://github.com/eclipse-syson/syson/issues/1120[#1120] [general-view] Add a graphical edge to represent the composition between `OccurrenceDefinition` and its `OccurrenceUsages`.
 - https://github.com/eclipse-syson/syson/issues/1125[#1125] [general-view] Simplify "Done" and "Start" actions
 - https://github.com/eclipse-syson/syson/issues/1131[#1131] [diagrams] Allow `NamespaceImport` graphical node labels to be wrapped to handle long names more easily.
+- https://github.com/eclipse-syson/syson/issues/1143[#1143] [export] Implement export of Decision Transition Usages.
```

Comment:
Please prefer "textual export" to "export" (same for import).
Please put "`" around SysMLv2 concepts (Decision Transition Usages => `Decision Transition Usages`).
This is true for every future contribution.

Reply:
"Decision Transition Usages" is not a real concept SysMLV2. Its a `TransistionUsage` used in the context of Action.

But ok for the phrasing of export.

Reply:
Yes I know. But Decision is a concept and Transition Usage too.

Reply:
Nop its `DecisionNode` and `TransitionUsage` and the term "Decision Transition Usages" is used directly in the specification.

Reply:
yes I should have written DecisionNode but you've got the idea.